causal-inference-for-the-brave-and-true/23-Challenges-with-Effect-Heterogeneity-and-Nonlinearity.ipynb

```diff
@@ -430,15 +430,15 @@
+    "Since `estimated_income` is highly predictive of conversion, we can say that customers with different `estimated_income` are located in different places along the S-shaped conversion curve. Customers with very high or very low `estimated_income` fall at the extremes of the curve, where the derivative is lower, meaning that increasing conversion is harder, which in turns means that the treatment effect is likely to be small. On the other hand, customers with reported income in the middle of the range also fall in the middle of the conversion curve, where the derivative is higher and, hece, the treatment effect will likely also be higher. I say likely because, in theory, it is possible for a variable to have such a strong effect modification force that it dominates the change in derivative we see as we traverse the conversion curve. However, at least from my experience, the curvature of the S shaped conversion tends to dominate every other effect modification we have. \n",
     " \n",
-    "This is not just me, though. Here is a slide I got from Susan Atheys' presentation for the Columbia Data Science Institute. Here, she is discussing the effect of a nudge to get students to apply for federal financial aid in order to pay for college. It's also a conversion problem. What she finds is that the best strategy is to target those students that are already likely to convert. She also says it is a bad idea to target those with low probability of conversion\n",
+    "It's not just me, though. Here is a slide from Susan Atheys' presentation at the Columbia Data Science Institute. Here, she is discussing the effect of a nudge to get students to apply for federal financial aid in order to pay for college. It's also a conversion problem. What she finds is that the best strategy is to target those students that are already likely to convert. She also says it is a bad idea to target those with low probability of conversion\n",
     " \n",
     "![image.png](data/img/hte-binary-outcome/slide-susan-athey.png)\n",
     " \n",
-    "Wait a minute! But that is not what you first said! You said that both very high and very low conversions have low derivative and hence, low treatment effect!\n",
+    "Wait a minute! That is not what you said above! You said that treatment effects were low in the middle, away from the low and the high end!\n",
```

Comment:
Its the other way around.
You said that treatment effects were low away from the middle, at the low and the high end!

Reply:
Fixed